2taesung/src/components/Pagination.tsx

```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link';
+
+export default function Pagination({ pageList }: { pageList: Array<number> }) {
+  return (
+    <ul className="pagination">
+      {pageList.map(num => {
```

Comment:
page라고 해주면 더 명확하게 느껴지지 않을까요?

```suggestion
      {pageList.map(page => {
```

Reply:
> page라고 해주면 더 명확하게 느껴지지 않을까요?

의견 감사합니다!
이건 pageList -> page가 자연스럽긴 한데 
실제로 num이 적절한 item의 변수가 될 것 같아서 ....
pageList를 더 적절한 변수명으로 바꿔줘야 좋을 것 같다는 생각이 드네요!!

Reply:
네~ 저도 애초에 pageList라는 변수명이 애매해서 생긴 문제라고 생각해요~